orchagent/main.cpp

```diff
@@ -158,10 +158,9 @@ bool getCfgSwitchType(DBConnector *cfgDb, string &switch_type)
     if (switch_type != "voq" && switch_type != "fabric" && switch_type != "switch")
     {
         cout << "Invalid switch type " << switch_type.c_str() << " configured";
```

Comment:
Where does the output of `cout ` go? To /var/log/syslog?

Reply:
This goes to /var/log/syslog since we call this after the logger system initialized. I'll change it  SWSS_LOG_ERROR() to make it clear.

Reply:
Fixed